[2021-08-16 15:47:54,717] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.delete_entry 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:54,752] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.delete_entry 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:54,753] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:54,754] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-08-16 15:47:54,754] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:54,774] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): delete_entry> on 2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:54,785] {standard_task_runner.py:52} INFO - Started process 508 to run task
[2021-08-16 15:47:54,799] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_complex', 'delete_entry', '2021-08-16T15:43:47.414310+00:00', '--job-id', '70', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpysi4pu6y', '--error-file', '/tmp/tmpv06ymlrc']
[2021-08-16 15:47:54,801] {standard_task_runner.py:77} INFO - Job 70: Subtask delete_entry
[2021-08-16 15:47:54,900] {logging_mixin.py:104} INFO - Running <TaskInstance: example_complex.delete_entry 2021-08-16T15:43:47.414310+00:00 [running]> on host 8f195762143d
[2021-08-16 15:47:55,026] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=delete_entry
AIRFLOW_CTX_EXECUTION_DATE=2021-08-16T15:43:47.414310+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:55,046] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-08-16 15:47:55,055] {bash.py:158} INFO - Running command: echo delete_entry
[2021-08-16 15:47:55,123] {bash.py:169} INFO - Output:
[2021-08-16 15:47:55,272] {bash.py:173} INFO - delete_entry
[2021-08-16 15:47:55,294] {bash.py:177} INFO - Command exited with return code 0
[2021-08-16 15:47:55,682] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=delete_entry, execution_date=20210816T154347, start_date=20210816T154754, end_date=20210816T154755
[2021-08-16 15:47:56,881] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-16 15:47:56,910] {local_task_job.py:146} INFO - Task exited with return code 0
